### PR TITLE
update version for scipy

### PR DESCRIPTION
**Pull request by suliuzh**
# What does this PR do?
 updating version requirement for scipy in 'examples\\distillation\\requirements.txt'.
fix [https://github.com/huggingface/transformers/issues/7967](url)

### CODE REVIEW

**Comment by LysandreJik** [comment]
Let's wait for Victor to answer on that issue before merging.

**Review by VictorSanh — APPROVED** [review]

**Comment by VictorSanh** [comment]
looks good!

**Review by LysandreJik — APPROVED** [review]

**Pull request merged**
